scripts/datasets/general_nlp_benchmark/prepare_glue.py

```diff
@@ -467,20 +467,20 @@ def read_broadcoverage_diagnostic(dir_path):
 
 
 GLUE_TASK2PATH = {
-    "cola": "https://firebasestorage.googleapis.com/v0/b/mtl-sentence-representations.appspot.com/o/data%2FCoLA.zip?alt=media&token=46d5e637-3411-4188-bc44-5809b5bfb5f4",  # noqa
-    "sst": "https://firebasestorage.googleapis.com/v0/b/mtl-sentence-representations.appspot.com/o/data%2FSST-2.zip?alt=media&token=aabc5f6b-e466-44a2-b9b4-cf6337f84ac8",  # noqa
+    "cola": "https://gluonnlp-numpy-data.s3-accelerate.amazonaws.com/datasets/text_classification/glue_superglue/glue/cola.zip",  # noqa
+    "sst": "https://gluonnlp-numpy-data.s3-accelerate.amazonaws.com/datasets/text_classification/glue_superglue/glue/sst.zip",  # noqa
```

Comment:
for our own mirrors, I recommend to always follow the naming convention where a content hash prefix is added as a suffix to the base name. The reason is that this will ensure updating data files will not affect results of older scripts.

Reply:
in the current setting where only hash verification happens, we won't be able to update the files without making files that contain old content hashes unrunnable.

Reply:
I think these are fixed and will not change (Because these are GLUE + SuperGLUE datasets). It should be fine to not add the hash prefix for these files.